cmd/mirror_comparison/main.go

```diff
@@ -13,10 +22,37 @@ func main() {
 		log.Fatal().Err(err).Msg("Error parsing log level")
 	}
 
-	_, err = config.NewMirrorComparisonConfig()
+	cfg, err := config.NewMirrorComparisonConfig()
 	if err != nil {
 		log.Fatal().Err(err).Msg("Error parsing config")
 	}
 
+	awsCfg, err := awsConfig.LoadDefaultConfig(context.TODO())
```

Comment:
`context.Background()` is a better fit here.

> TODO returns a non-nil, empty [Context](https://pkg.go.dev/context#Context). Code should use context.TODO when it's unclear which Context to use or it is not yet available (because the surrounding function has not yet been extended to accept a Context parameter).